bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/BlueGigaBluetoothDevice.java

```diff
@@ -229,27 +236,18 @@ public void bluegigaEventReceived(BlueGigaResponse event) {
             }
 
             if (connectionState == ConnectionState.DISCOVERING) {
-                // We want to wait for an advertisement and a scan response before we call this discovered.
-                // The intention is to gather a reasonable amount of data about the device given devices send
-                // different data in different packets...
-                // Note that this is possible a bit arbitrary and may be refined later.
```

Comment:
I removed waiting for a `SCAN_RESPONSE`, as not all devices do send a scan response at all (e.g. EnOcean PTM215B) which completely blocks discovery.  I don't really know for which use case it was added, so I'm not sure what would be a better implementation. I think Bluez has no such thing.